### ISSUE TITLE
Error : No peer answered in community

### ISSUE THREAD

**Issue by vincentux** [duniter/sakia#648]
![selection_061](https://cloud.githubusercontent.com/assets/11737560/24576421/fa9e34b8-16bb-11e7-813f-c90cfbe51788.jpg)

**Comment by vincentux** [comment]
Et quand j'ai fermé la fenêtre : 
`QXcbConnection: XCB error: 3 (BadWindow), sequence: 1751, resource id: 9295222, major code: 40 (TranslateCoords), minor code: 0`

**Comment by jytou** [comment]
Same thing for me. Maybe you should add a few more "stable" peers so that at least one of them will answer. But the best would be for us to be able to manually type one if we know one. Because at this point I'm completely stuck and I cannot go further, which is quite troubling for a newcomer.

**Comment by galuel** [comment]
Yes there should be available in advanced settings the possibility to update manually the list of peers to connect...

**Comment by jpfox** [comment]
Hi
I've installed sakia 0.31 on debian Jessie and got the same message when creating an identity

    Error : No peer answered in community

Is there something to do to permit peer connection : port redirection on dsl box or something else ?

**Comment by Insoleet** [comment]
You can edit the root_servers.yml file in the installation directory under sakia dir to add servers manually.

Can you upload your g1.db file from the directory `.config/sakia/Default Profile` ?

**Comment by M5oul** [comment]
> You can edit the root_servers.yml file in the installation directory under sakia dir to add servers manually.

I don't think it's possible with the build.

**Comment by jpfox** [comment]
Which peer can I add ??
I can't join my file just now.
If there is a record to add in slite db, I can do it

**Comment by Insoleet** [comment]
@Moul : yes it is. In (sakia install dir)/sakia you will find the file.

**Comment by M5oul** [comment]
In fact, I didn't went enough far.
`sakia_path/sakia/sakia/root_servers.yml`.

**Comment by M5oul** [comment]
@jpfox,
You could try nodes on http://g1.duniter.org:10901/network/peers

**Comment by jytou** [comment]
Actually g1.duniter.org is on port 10901 and the file specifies 80 instead. :)

**Comment by M5oul** [comment]
It works fine on port 80:
```bash
curl g1.duniter.org:80
{
  "duniter": {
    "software": "duniter",
    "version": "1.1.0",
    "forkWindowSize": 100
  }
}⏎                    
```

**Comment by jytou** [comment]
I've tried adding some other peers, it still gives the same error message.
The interesting part is that when I close the window, I get Sakia's main window which shows all the peers on the network (they appear in red and orange, though). So it does manage to get the list, but doesn't want to connect with my identity. I double-checked that my public key is the correct one.

**Comment by jytou** [comment]
And I have errors in the console:

    ERROR:root:Unclosed client session
    client_session: <aiohttp.client.ClientSession object at 0x7f0eb03bbef0>

**Comment by jytou** [comment]
Could it be related to the fact that I don't have IPv6 and can only connect through IPv4?

**Comment by M5oul** [comment]
>Could it be related to the fact that I don't have IPv6 and can only connect through IPv4?


`curl -4 g1.duniter.org:80` works fine on my side.

**Comment by jytou** [comment]
> curl -4 g1.duniter.org:80 works fine on my side.
I do access g1.duniter, I was just wondering if sakia had a problem with ipv4 in the new version, which would explain why it works for some and not for others.

**Comment by Insoleet** [comment]
I need one of your g1.db file to analyze the problem.

**Comment by Insoleet** [comment]
Bug fixed in 0.31.1.
